server/db/model/sync/server/server.js

```diff
@@ -0,0 +1,33 @@
+const Server = require('../../code_snippets/model-server');
+
+console.log('hello from server sync', typeof Server);
+const serverSeed = async () => {
+  const S1 = await Server.create({
+    id: 'S1',
+    title: 'Server: imports',
+    snippet: `
+    Const express = require(‘express’)
+    Const app = express()
+    Const PORT = process.env.PORT || 3000
+    `,
+  });
+
+  const S2 = await Server.create({
+    id: 'S2',
+    title: 'Server: initializeApp',
+    snippet: `
+    Const initializeApp = async () => {
+      Try {
+        app.listen(PORT, () => console.log(‘app is listening on $ {PORT}
```

Comment:
need to add  close console.log parentheses  at the end of line 21

Reply:
This has been fixed